crates/bevy_dev_tools/src/lib.rs

```diff
@@ -45,3 +48,110 @@ impl Plugin for DevToolsPlugin {
         }
     }
 }
+
+/// Unique identifier for [`DevTool`].
+#[derive(Debug, Hash, Eq, PartialEq, Clone)]
+pub struct DevToolId(pub Uuid);
```

Comment:
I'm wondering: Why not use a TypeId map instead of Uuid? I think its much more simple.

Reply:
Do we want to identify each dev tool with a different type? We could use plugin struct for it, but what if plugin adds more than one dev tool?

Reply:
I was expecting to identify every tool with a single config type registered to it, but I see that's a similar approach taken by the gizmos multiple configs, and we aren't storing configs in this map by now, so, with we have now on this PR, the TypeId doesn't make sense.
I can see number of reasons why offering a config type though. For example, we might want to do more than enable and disable things, we might want this DevToolsStore to store entire configurations in the future, like storing the FpsOverlayConfig here, and let the user refer to it from the TypeId (which can be easily created and so).